### DISCUSSION TITLE
subcommand or option?

### DISCUSSION THREAD

**Discussion by chetan0402** [tldr-pages/tldr#9920] [Q&A] [answered]
so there is this one windows command im thinking to work on but it has the "subcommands" like command /subcommand so should i just make the .md as command-subcommand or treat as an option and create a command.md and add all the other "subcommands" in it

**Comment by navarroaxel** [comment, marked as answer]
you can create pages for subcommands following the `command-subcommand.md` format for the filename, but you can create the main page first.
